fe/fe-core/src/main/java/org/apache/doris/qe/Coordinator.java

```diff
@@ -841,6 +842,7 @@ private void sendPipelineCtx() throws TException, RpcException, UserException {
                         beToPipelineExecCtxs.putIfAbsent(pipelineExecContext.backend.getId(), ctxs);
                     }
                     ctxs.addContext(pipelineExecContext);
+                    ctxs.setInstanceNumber(entry.getValue().getFragmentNumOnHost());
```

Comment:
`instanceNumber` should be a const value and set in ctor

Reply:
`instanceNumber` is not `parallel_pipeline_task_num` but the actual num of fragments for each BE. Can we get it in c'tor?